src/logic/operate.js

```diff
@@ -0,0 +1,31 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  let result = 0;
+  const firstNumber = Big(numberOne);
+  const secondNumber = Big(numberTwo);
+
+  if (operation === '+') {
+    result = firstNumber.plus(secondNumber);
+  }
+
+  if (operation === '-') {
+    result = firstNumber.minus(secondNumber);
+  }
+
+  if (operation === 'x') {
+    result = firstNumber.times(secondNumber);
+  }
+
+  if (operation === '÷') {
+    result = firstNumber.div(secondNumber);
+  }
+
+  if (operation === '%') {
+    result = firstNumber.mod(secondNumber);
+  }
+
```

Comment:
**[OPTIONAL]**
* [ ] You could add '-/+' modifier to change numbers from negative to positive, vice versa.🤗